publication-commons/src/test/java/no/unit/nva/publication/service/impl/ResourceServiceTest.java

```diff
@@ -196,23 +195,6 @@ void shouldInstantiateResourceServiceForProvidedTable() {
         assertThat(tableNames, hasItem(customTable));
     }
 
-    @Test
```

Comment:
Removed test and method in resource service it was testing - method is never used in production.